test/basic.js

```diff
@@ -7,27 +7,23 @@ var tap = require('tap')
 var globalBefore = Object.keys(global)
 var mm = require('../')
 var patterns = require('./patterns.js')
-var regexps = patterns.regexps
-var re = 0
 
 tap.test('basic tests', function (t) {
   var start = Date.now()
 
-  // [ pattern, [matches], MM opts, files, TAP opts]
   patterns.forEach(function (c) {
-    if (typeof c === 'function') return c()
     if (typeof c === 'string') return t.comment(c)
 
-    var pattern = c[0],
```

Comment:
If you read http://standardjs.com/rules.html, you'll see a section that says this:

> For var declarations, write each declaration in its own statement.
> 
> ```
> // ✓ ok 
> var silent = true
> var verbose = true
>  
> // ✗ avoid 
> var silent = true, verbose = true
>  
> // ✗ avoid 
> var silent = true,
>     verbose = true
> ```

...But IIRC I did that so it would match the format of `test/defaults.js` to make it more obvious that they follow the same pattern.